app/src/main/java/com/beemdevelopment/aegis/util/IOUtils.java

```diff
@@ -0,0 +1,37 @@
+package com.beemdevelopment.aegis.util;
+
+import java.io.ByteArrayOutputStream;
+import java.io.DataInputStream;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+public class IOUtils {
+    private IOUtils() {
```

Comment:
Empty constructor, looks like it can be removed.

Reply:
This private constructor is there to that the class can't be instantiated, as it only contains static methods.

Reply:
TIL Java doesn't support static classes.